### PR TITLE
Drivers: add latency log for lidar

### PR DESCRIPTION
**Pull request by fengqikai1414**

### CODE REVIEW

File: modules/drivers/velodyne/compensator/compensator_component.cc [hunk 1]
```diff
@@ -62,10 +62,14 @@ bool CompensatorComponent::Proc(
   }
   point_cloud_compensated->Clear();
   if (compensator_->MotionCompensation(point_cloud, point_cloud_compensated)) {
-    uint64_t diff = cyber::Time().Now().ToNanosecond() - start;
-    AINFO << "compenstator diff:" << diff
+    auto now = cyber::Time().Now().ToNanosecond();
```
Comment (Capri2014) [inline modules/drivers/velodyne/compensator/compensator_component.cc]: use `const` here?

File: modules/drivers/velodyne/fusion/pri_sec_fusion_component.cc [hunk 2]
```diff
@@ -64,6 +64,9 @@ bool PriSecFusionComponent::Proc(
     }
     std::this_thread::sleep_for(std::chrono::milliseconds(5));
   }
+  auto diff = Time::Now().ToNanosecond() -
+              static_cast<uint64_t>(target->header().lidar_timestamp());
```
Comment (xiaoxq) [inline modules/drivers/velodyne/fusion/pri_sec_fusion_component.cc]: Seems like that it is `uint64_t`. So the `static_cast` is not needed.

Comment (fengqikai1414) [reply modules/drivers/velodyne/fusion/pri_sec_fusion_component.cc]: Yes, you are right. google::protobuf::uint64 is an alias for uint64_t. Have fixed.

File: modules/drivers/velodyne/compensator/compensator_component.cc (hunk 1 above)
Comment (fengqikai1414) [reply modules/drivers/velodyne/compensator/compensator_component.cc]: Fixed.

**Pull request merged**